src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -132,7 +142,7 @@ public Biome(ModElement element) {
 	@Override public BufferedImage generateModElementPicture() {
 		return MinecraftImageGenerator.Preview
 				.generateBiomePreviewPicture(getModElement().getWorkspace(), airColor, grassColor, waterColor,

```

Comment:
Pass treeType 1 if custom trees are enabled, otherwise 0 if trees list has at least one element, and pass treesPerChunk as 0 if nor vanilla nor custom trees are enabled, to disable drawing trees in such case.

Reply:
I'm not sure if what I made is what you tried to say, but I will change if it's not correct.

Reply:
This part is still missing: and pass treesPerChunk as 0 if nor vanilla nor custom trees are enabled, to disable drawing trees in such case. - this way tree won't be drawn if no tree would be defined but treesPerChunk is non-zero.